src/test/java/io/fabric8/vertx/maven/plugin/AbstractTestCase.java

```diff
@@ -0,0 +1,25 @@
+package io.fabric8.vertx.maven.plugin;
+
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.codehaus.plexus.util.ReaderFactory;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * @author kameshs
+ */
+public class AbstractTestCase {
+
+    protected Model buildModel(File pomFile) {
```

Comment:
Just a question of taste but as it does not use any field, could it be static, avoiding the "inheritance" pattern.